docs/oihPostmanGuide.md

```diff
@@ -0,0 +1,422 @@
+# Introduction
+
+This document is designed as a guide on _test the open integration hub with postman_.
+
+- [Introduction](#introduction)
+  - [Prerequisites](#prerequisites)
+  - [SetUp](#setup)
+  - [Use Cases](#use-cases)
+    - [Flows](#flows)
+      - [Get All Flows](#get-all-flows)
+      - [Get a Flow](#get-a-flow)
+      - [Post a Flow](#post-a-flow)
+      - [Update a Flow](#update-a-flow)
+      - [Delete a Flow](#delete-a-flow)
+      - [Start a Flow](#start-a-flow)
+      - [Stop a Flow](#stop-a-flow)
+    - [Components](#components)
+      - [Get All Components](#get-all-components)
+      - [Get a Component](#get-a-component)
+      - [Post a Component](#post-a-component)
+      - [Update a Component](#update-a-component)
+      - [Delete a Component](#delete-a-component)
+    - [Identity and Access Management](#identity-and-access-management)
+      - [Login](#login)
+      - [Get all Users](#get-all-users)
+      - [Add a new User](#add-a-new-user)
+      - [Delete a User](#delete-a-user)
+      - [Update a User](#update-a-user)
+  - [Examples](#examples)
+    - [Flow Examples](#flow-examples)
+    - [Component Examples](#component-examples)
+
+## Prerequisites
+
+In order to use the Open Integration Hub we provided a demo environment and a [postman collection](postman/OIH_Framework_Showcase.postman_collection.json) and a [postman environment](OIH_Framework.postman_environment.json) to test the functionalities of the Open Integration Hub.
+
+We suggest to download and import these files to easily process the steps described in the following.
+
+## SetUp
+
+Before you start using the framework you need a user account (username & passoword). Please request a new account from [@philecs](https://github.com/philecs). E-Mail: <philipp.hoegner@cloudecosystem.org>.
+
+It is necessary to add valid user data in order to perform the requests.
+Please process the following steps in order to be able to generate a valid token for the requests:
+
+1. Right click on the `OIH_SHOWCASE_COLLECTION` and click on `edit` OR click on the three dots `...` and click `edit`.
+2. Jump to `Pre-request Scripts`
+3. Replace `{YOUR_USERNAME}` with the username you received
+4. Replace `{YOUR_PASSWORD}` with the password you received
+5. Click on `Update`
+
```

Comment:
Is this easier?

const username = 'franz.degenhardt@x-integrate.com';
const password = 'xyz123;
const echoPostRequest = {
  url: `http://iam.openintegrationhub.com/login?username=${username}&password=${password}`,
  method: 'POST',
  header: 'Content-Type:application/json'
};

Reply:
Good point. I'll update this.

Reply:
Done